### DISCUSSION TITLE
官方固件的源里安装passwall，没有ipt2socks

### DISCUSSION THREAD

**Discussion by xiaoyaoyuxin** [immortalwrt/immortalwrt#1166] [Q&A]
X86的软路由，安装的官方编译的固件，在官方软件包源里安装passwall，提示没有ip2socks

求指点

![ip2socks](https://github.com/immortalwrt/immortalwrt/assets/8025850/2d3920c3-602e-4196-879d-4da4fbc152f9)

**Comment by shawn996** [comment]
[ipt2socks_1.1.3-1_x86_64.zip](https://github.com/immortalwrt/immortalwrt/files/14321422/ipt2socks_1.1.3-1_x86_64.zip)
是呢，不知道为啥少了x86的版本

**Comment by WROIATE** [comment]
imm用的应该是nft，这玩意是针对iptables的？

**Reply by ghwwx** [reply]
那么imm用什么科学上网插件呢？

**Reply by sys-default** [reply]
> 那么imm用什么科学上网插件呢？

https://github.com/daeuniverse

可能试试大鹅吧，运行在内核上的，天灵大佬应该也是这个项目的开发人员之一
imm应用库里有 [luci-app-daed](https://github.com/QiuSimons/luci-app-daed)

**Reply by ghwwx** [reply]
谢谢！另外 最新版的Imm的 softethervpn不知道为啥总是不生效，listening的ip也改为0.0.0.0了。奇怪。
